src/Builder/FrontendDialectTransformer.cpp

```diff
@@ -287,7 +288,8 @@ class FrontendGenImpl {
    * Import an onnx tensor type by determining and returning its type
    * @param type_proto onnx tensor TypeProto.
```

Comment:
please add a @param description of dimParam

Reply:
Done. Thanks!